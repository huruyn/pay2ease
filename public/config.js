window.API_URL = "https://zippy-retail-api-dev.azurewebsites.net";