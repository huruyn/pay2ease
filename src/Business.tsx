import { useMsal } from '@azure/msal-react';
import { useTranslation, Trans } from 'react-i18next';
import Circle_White_01 from './assets/img/background/Circle_White-01.jpg';
import ZippyCash_Icons_K from './assets/img/roundicons/business/ZippyCash_Icons_K.png';
import ZippyCash_Icons_L from './assets/img/roundicons/business/ZippyCash_Icons_L.png';
import ZippyCash_Icons_M from './assets/img/roundicons/business/ZippyCash_Icons_M.png';
import BusinessBg from './assets/img/background/help_my_bus_hero.jpg';
import BlueCircle from './assets/img/roundicons/business/bluecircle.png';
import NumberCircle1 from './assets/img/roundicons/business/numbercircle_1.png';
import NumberCircle2 from './assets/img/roundicons/business/numbercircle_2.png';
import NumberCircle3 from './assets/img/roundicons/business/numbercircle_3.png';
import NumberCircle4 from './assets/img/roundicons/business/numbercircle_4.png';
import { b2cPolicies, loginRequest } from './authConfig';
import Footer from './footer/Footer';

export default function Business() {
  const { instance } = useMsal();
  const { t } = useTranslation();

  return (
    <>
      <div
        className="bg-light"
        style={{
          backgroundImage: `url(${Circle_White_01})`,
          backgroundSize: '146.5%',
        }}
      >
        <div className="container">
          <div className="row py-5">
            <div className="col">
              <div className="mb-3">
                <h2 style={{ color: '#FF1926' }}>
                  {t('business.business_title')}
                </h2>
                <h1
                  className="redtitle hero-heading display-3"
                  style={{ fontSize: '4em', paddingBottom: '20px' }}
                >
                  {t('business.header_desc')}
                </h1>
                <button
                  type="button"
                  className="btn rounded-0 my-buttonred"
                  onClick={() => instance.loginRedirect({
                    ...loginRequest,
                    authority: b2cPolicies.authorities.signUpBusiness.authority,
                  })}
                >
                  {t('business.sign_up_btn_title')}
                </button>
              </div>
            </div>
          </div>
        </div>
        <div>
          <div className="container">
            <div className="row align-items-center py-4">
              <div className="col-12 col-md-3">
                <img
                  src={ZippyCash_Icons_K}
                  alt="an illustration of a woman holding dollar signs"
                  className="img-fluid centerimage"
                  data-aos="flip-left"
                />
              </div>
              <div className="col-12 col-md-9">
                <p className="m-0 content-home">
                  <Trans i18nKey="business.business_description_1" />
                </p>
              </div>
            </div>
          </div>
        </div>
        <div id="bluegradient" className="bg-light">
          <div className="container">
            <div className="row align-items-center py-4  flex-md-row-reverse">
              <div
                className="col-12 col-md-3 text-end position-relative"
                data-aos="flip-left"
              >
                <img
                  src={ZippyCash_Icons_L}
                  alt="illustrated of young people all touching a large light bulb"
                  className="img-fluid position-absolute top-0 start-50 translate-middle "
                />
              </div>
              <div className="col-12 col-md-9 circleoverlap">
                <p className="m-0 whitecopy">
                  <Trans i18nKey="business.business_description_2" />
                </p>
              </div>
            </div>
          </div>
        </div>
        <div style={{ backgroundColor: '#EDF5F5' }}>
          <div className="container">
            <div className="row align-items-center py-4">
              <div className="col-12 col-md-3">
                <img
                  src={ZippyCash_Icons_M}
                  alt="illustrated of a young woman jumping in the air"
                  className="img-fluid centerimage"
                  data-aos="flip-left"
                />
              </div>
              <div className="col-12 col-md-9">
                <p className="m-0 content-home">
                  <Trans i18nKey="business.business_description_3" />
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div style={{ backgroundColor: 'white' }}>
        <div className="container">
          <div className="row pt-5 pb-4">
            <div className="col">
              <div className="mb-3">
                <h1
                  className="redtitle hero-heading display-3"
                  style={{ fontSize: '4em' }}
                >
                  {t('business.how_can_help')}
                </h1>
              </div>
            </div>
          </div>
          <div className="row">
            <div className="col-12 col-md-5 col-lg-4">
              <h4 className="redtitle">{t('business.integrate_payment')}</h4>
              <p className="content-home">
                {t('business.integrate_payment_desc')}
              </p>
              <br />
              <h4 className="redtitle">{t('business.send_and_accept_money')}</h4>
              <p className="content-home">
                {t('business.send_and_accept_money_desc')}
              </p>
              <br />
              <h4 className="redtitle">{t('business.make_payment')}</h4>
              <p className="content-home">
                {t('business.make_payment_desc')}
              </p>
              <br />
              <h1 className="redtitle mb-5">{t('business.signup_free')}</h1>
            </div>
            <div
              className="col-12 col-md-7 col-lg-8 backgroundimage"
              style={{
                backgroundImage: `url(${BusinessBg})`,
                backgroundSize: 'cover',
              }}
            />
          </div>
        </div>
      </div>
      <main style={{ backgroundColor: '#51aad8' }}>
        <div className="container">
          <div className="row">
            <div className="col-12 col-md-6 col-lg-3">
              <div
                className="position-relative top-0"
                data-aos="flip-left"
                data-aos-delay="0"
              >
                <img
                  src={BlueCircle}
                  alt=""
                  className="img-fluid centerimage bluecircle"
                />
                <img
                  src={NumberCircle1}
                  alt=""
                  className="position-absolute top-0 start-50 numbercicle"
                />

                <div className="position-absolute halfcirclecopy">
                  <h5 className="whitetitle text-center">
                    <a href="business-signup.html" className="whitetitle">
                      <Trans i18nKey="business.signup_here" />
                    </a>
                  </h5>
                </div>
              </div>
            </div>
            <div className="col-12 col-md-6 col-lg-3">
              <div
                className="position-relative top-0"
                data-aos="flip-left"
                data-aos-delay="200"
              >
                <img
                  src={BlueCircle}
                  alt=""
                  className="img-fluid centerimage bluecircle"
                />
                <img
                  src={NumberCircle2}
                  alt=""
                  className="position-absolute top-0 start-50 numbercicle"
                />

                <div className="position-absolute halfcirclecopy">
                  <h5 className="whitetitle text-center">
                    <Trans i18nKey="business.send_text_email" />
                  </h5>
                  <p className="whitecopy text-center">
                    <Trans i18nKey="business.choose_works" />
                  </p>
                </div>
              </div>
            </div>
            <div className="col-12 col-md-6 col-lg-3">
              <div
                className="position-relative top-0"
                data-aos="flip-left"
                data-aos-delay="400"
              >
                <img
                  src={BlueCircle}
                  alt=""
                  className="img-fluid centerimage bluecircle"
                />
                <img
                  src={NumberCircle3}
                  alt=""
                  className="position-absolute top-0 start-50 numbercicle"
                />

                <div className="position-absolute halfcirclecopy">
                  <h5 className="whitetitle text-center">
                    <Trans i18nKey="business.link_download" />
                  </h5>
                  <p className="whitecopy text-center">
                    <Trans i18nKey="business.link_download_desc" />
                  </p>
                </div>
              </div>
            </div>
            <div className="col-12 col-md-6 col-lg-3">
              <div
                className="position-relative top-0"
                data-aos="flip-left"
                data-aos-delay="600"
              >
                <img
                  src={BlueCircle}
                  alt=""
                  className="img-fluid centerimage bluecircle"
                />
                <img
                  src={NumberCircle4}
                  alt=""
                  className="position-absolute top-0 start-50 numbercicle"
                />

                <div className="position-absolute halfcirclecopy">
                  <h5 className="whitetitle text-center">
                    <Trans i18nKey="business.open_and_start" />
                  </h5>
                </div>
              </div>
            </div>
          </div>
        </div>
      </main>
      <Footer />
    </>
  );
}
