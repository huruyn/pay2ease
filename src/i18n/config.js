import i18n from 'i18next';
import { initReactI18next } from 'react-i18next';

i18n.use(initReactI18next).init({
  fallbackLng: 'en-US',
  lng: 'en-US',
  resources: {
    'en-CA': {
      translations: require('./locales/en-CA/translations.json')
    },
    'en-US': {
      translations: require('./locales/en-US/translations.json')
    },
    'es-US': {
      translations: require('./locales/es-US/translations.json')
    },
    'fr-CA': {
      translations: require('./locales/fr-CA/translations.json')
    },
  },
  ns: ['translations'],
  defaultNS: 'translations'
});

i18n.languages = ['en-CA', 'en-US', 'es-US', 'fr-CA'];

export default i18n;