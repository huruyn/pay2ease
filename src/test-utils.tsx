import { FC, ReactElement } from 'react';
import { render, RenderOptions } from '@testing-library/react';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import './i18n/config';
import { store } from './app/store';

const AllTheProviders: FC = ({ children }) => (
  <Provider store={store}>
    <BrowserRouter>{children}</BrowserRouter>
  </Provider>
);

const customRender = (
  ui: ReactElement,
  options?: Omit<RenderOptions, 'wrapper'>,
) => render(ui, { wrapper: AllTheProviders, ...options });

export * from '@testing-library/react';
export { customRender as renderWithProviders };
