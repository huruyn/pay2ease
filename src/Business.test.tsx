import { renderWithProviders, fireEvent } from './test-utils';
import './i18n/config';

import Business from './Business';

const ReactTestRenderer = require('react-test-renderer');

const mockLoginRedirect = jest.fn();
jest.mock('@azure/msal-react', () => ({
  ...(jest.requireActual('@azure/msal-react') as any),
  useMsal: () => ({
    instance: {
      loginRedirect: mockLoginRedirect,
    },
  }),
}));

describe('Business Component', () => {
  it('matches the snapshot', () => {
    const tree = ReactTestRenderer.create(<Business />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('click a button on business', () => {
    const { container } = renderWithProviders(<Business />);

    fireEvent.click(container.querySelectorAll('button')[0]);
    expect(mockLoginRedirect).toHaveBeenCalled();
  });
});
