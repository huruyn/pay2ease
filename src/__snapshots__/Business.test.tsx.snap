// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Business Component matches the snapshot 1`] = `
Array [
  <div
    className="bg-light"
    style={
      Object {
        "backgroundImage": "url(Circle_White-01.jpg)",
        "backgroundSize": "146.5%",
      }
    }
  >
    <div
      className="container"
    >
      <div
        className="row py-5"
      >
        <div
          className="col"
        >
          <div
            className="mb-3"
          >
            <h2
              style={
                Object {
                  "color": "#FF1926",
                }
              }
            >
              business account
            </h2>
            <h1
              className="redtitle hero-heading display-3"
              style={
                Object {
                  "fontSize": "4em",
                  "paddingBottom": "20px",
                }
              }
            >
              Make your money work your way!
            </h1>
            <button
              className="btn rounded-0 my-buttonred"
              onClick={[Function]}
              type="button"
            >
              Sign up for a business account
            </button>
          </div>
        </div>
      </div>
    </div>
    <div>
      <div
        className="container"
      >
        <div
          className="row align-items-center py-4"
        >
          <div
            className="col-12 col-md-3"
          >
            <img
              alt="an illustration of a woman holding dollar signs"
              className="img-fluid centerimage"
              data-aos="flip-left"
              src="ZippyCash_Icons_K.png"
            />
          </div>
          <div
            className="col-12 col-md-9"
          >
            <p
              className="m-0 content-home"
            >
              No matter how big or small your business is, everyone is in the global market. That's why you need the best rates and lowest fees for you brand. Zippy Cash gives you the best rates available every time and takes the complexity out of international commerce. Our fees can't be beat.
            </p>
          </div>
        </div>
      </div>
    </div>
    <div
      className="bg-light"
      id="bluegradient"
    >
      <div
        className="container"
      >
        <div
          className="row align-items-center py-4  flex-md-row-reverse"
        >
          <div
            className="col-12 col-md-3 text-end position-relative"
            data-aos="flip-left"
          >
            <img
              alt="illustrated of young people all touching a large light bulb"
              className="img-fluid position-absolute top-0 start-50 translate-middle "
              src="ZippyCash_Icons_L.png"
            />
          </div>
          <div
            className="col-12 col-md-9 circleoverlap"
          >
            <p
              className="m-0 whitecopy"
            >
              Setting up international bank accounts may sound daunting but with Zippy we deliver everything you need to start managing accounts abroad in minutes. How is it this easy? We have excellent partnerships with international financial brands, tech solutions that serve as an efficient connector, prioritizing ease, speed and security.
            </p>
          </div>
        </div>
      </div>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#EDF5F5",
        }
      }
    >
      <div
        className="container"
      >
        <div
          className="row align-items-center py-4"
        >
          <div
            className="col-12 col-md-3"
          >
            <img
              alt="illustrated of a young woman jumping in the air"
              className="img-fluid centerimage"
              data-aos="flip-left"
              src="ZippyCash_Icons_M.png"
            />
          </div>
          <div
            className="col-12 col-md-9"
          >
            <p
              className="m-0 content-home"
            >
              Paying invoices, people and employees just got zippier. We always give you the lowest transaction fees and make it easy (dare we say even fun?) to manage your payments.
              <br />
              Find out more here.
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    style={
      Object {
        "backgroundColor": "white",
      }
    }
  >
    <div
      className="container"
    >
      <div
        className="row pt-5 pb-4"
      >
        <div
          className="col"
        >
          <div
            className="mb-3"
          >
            <h1
              className="redtitle hero-heading display-3"
              style={
                Object {
                  "fontSize": "4em",
                }
              }
            >
              How can Zippy help my business?
            </h1>
          </div>
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="col-12 col-md-5 col-lg-4"
        >
          <h4
            className="redtitle"
          >
            Integrate Payment Systems
          </h4>
          <p
            className="content-home"
          >
            With Zippy, you can put everything in one place for easier accounting and financial management.
          </p>
          <br />
          <h4
            className="redtitle"
          >
            Send and Accept Money
          </h4>
          <p
            className="content-home"
          >
            Paying cross border is easier, cheaper and faster with Zippy. With or without a bank account, you can pay vendors and charge customers.
          </p>
          <br />
          <h4
            className="redtitle"
          >
            Make Payments a Revenue Stream
          </h4>
          <p
            className="content-home"
          >
            We offer consumers a 99 cent instant payout of points, or they can wait to get them for free. When consumers choose the faster payout when making a payment at your business, your business get a little bit back. That means payments won't cost you and can become a passive revenue stream.
          </p>
          <br />
          <h1
            className="redtitle mb-5"
          >
            Sign up free!
          </h1>
        </div>
        <div
          className="col-12 col-md-7 col-lg-8 backgroundimage"
          style={
            Object {
              "backgroundImage": "url(help_my_bus_hero.jpg)",
              "backgroundSize": "cover",
            }
          }
        />
      </div>
    </div>
  </div>,
  <main
    style={
      Object {
        "backgroundColor": "#51aad8",
      }
    }
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col-12 col-md-6 col-lg-3"
        >
          <div
            className="position-relative top-0"
            data-aos="flip-left"
            data-aos-delay="0"
          >
            <img
              alt=""
              className="img-fluid centerimage bluecircle"
              src="bluecircle.png"
            />
            <img
              alt=""
              className="position-absolute top-0 start-50 numbercicle"
              src="numbercircle_1.png"
            />
            <div
              className="position-absolute halfcirclecopy"
            >
              <h5
                className="whitetitle text-center"
              >
                <a
                  className="whitetitle"
                  href="business-signup.html"
                >
                  Sign up
                  <br />
                  here
                </a>
              </h5>
            </div>
          </div>
        </div>
        <div
          className="col-12 col-md-6 col-lg-3"
        >
          <div
            className="position-relative top-0"
            data-aos="flip-left"
            data-aos-delay="200"
          >
            <img
              alt=""
              className="img-fluid centerimage bluecircle"
              src="bluecircle.png"
            />
            <img
              alt=""
              className="position-absolute top-0 start-50 numbercicle"
              src="numbercircle_2.png"
            />
            <div
              className="position-absolute halfcirclecopy"
            >
              <h5
                className="whitetitle text-center"
              >
                We will send you a
                <br />
                 link via text or email
              </h5>
              <p
                className="whitecopy text-center"
              >
                (you choose which
                <br />
                 works for you)
              </p>
            </div>
          </div>
        </div>
        <div
          className="col-12 col-md-6 col-lg-3"
        >
          <div
            className="position-relative top-0"
            data-aos="flip-left"
            data-aos-delay="400"
          >
            <img
              alt=""
              className="img-fluid centerimage bluecircle"
              src="bluecircle.png"
            />
            <img
              alt=""
              className="position-absolute top-0 start-50 numbercicle"
              src="numbercircle_3.png"
            />
            <div
              className="position-absolute halfcirclecopy"
            >
              <h5
                className="whitetitle text-center"
              >
                Click the link to
                <br />
                 download the app
              </h5>
              <p
                className="whitecopy text-center"
              >
                Zippy is available on over
                <br />
                 400 smart devices
              </p>
            </div>
          </div>
        </div>
        <div
          className="col-12 col-md-6 col-lg-3"
        >
          <div
            className="position-relative top-0"
            data-aos="flip-left"
            data-aos-delay="600"
          >
            <img
              alt=""
              className="img-fluid centerimage bluecircle"
              src="bluecircle.png"
            />
            <img
              alt=""
              className="position-absolute top-0 start-50 numbercicle"
              src="numbercircle_4.png"
            />
            <div
              className="position-absolute halfcirclecopy"
            >
              <h5
                className="whitetitle text-center"
              >
                Open the app
                <br />
                 and start 
                <br />
                 Zipping!
              </h5>
            </div>
          </div>
        </div>
      </div>
    </div>
  </main>,
  <div
    className="footer-content-wrapper"
  >
    <div
      className="quick-links container"
    >
      <div
        className="row"
      >
        <h3
          className="col-sm-12"
        >
          Quick Links
        </h3>
      </div>
      <div
        className="row"
      >
        <div
          className="col-sm-2"
        >
          <ul>
            <li>
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Home
              </a>
            </li>
            <li>
              Please Review Our Legal Stuff by visiting our
               
              <a
                className="text-decoration-none"
                href="/legal"
                onClick={[Function]}
              >
                Privacy Policy | TOU | EULA | CardHolder T&C Pages
              </a>
            </li>
          </ul>
        </div>
        <div
          className="col-sm-2"
        >
          8465 W Sahara Ave. Las Vegas, NEVADA
          <ul>
            <li>
              <a
                href="tel:+17026231876"
              >
                1.702.623.1876
              </a>
            </li>
            <li>
              <a
                href="mailto:contact@zippy.cash"
              >
                contact@zippy.cash
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col-md-12 col-sm-12 col-xs-12 text-left"
        >
          © 2021 Zippy Cash LLC  All rights reserved.
          <br />
          <br />
          The Program Manager for the Zippy Cash® Prepaid Visa Debit Card is PayQuicker, LLC
          <br />
          Zippy Cash® is a registered trademark of Zippy Cash, LLC
          <br />
          The Zippy Cash Prepaid Visa Debit Card is issued by Metropolitan Commercial Bank (Member FDIC) pursuant to a license from Visa U.S.A. Inc. “Metropolitan Commercial Bank” and “Metropolitan” are registered trademarks of Metropolitan Commercial Bank ©2014. BY USING THIS CARD YOU AGREE TO THE TERMS AND CONDITIONS OF THE CARDHOLDER AGREEMENT AND FEE SCHEDULE, IF ANY. All trademarks and service marks are the property of their respective owners.
          <br />
          In compliance with the USA PATRIOT Act* and other laws, anyone who is a US resident over 18 with a valid Social Security number and a verifiable physical US address (no PO boxes or APO addresses) can apply for an Zippy Cash Card. This information must be provided, along with other information pertinent to receiving approval, when applying for a Zippy Cash Card. Currently, we do not issue the Zippy Cash Card to residents of Puerto Rico, Guam, the Virgin Islands, American Samoa, or other US territories.
          <br />
          *Important information for opening a card account: To help the federal government fight the funding of terrorism and money laundering activities, the USA PATRIOT Act requires all financial institutions and their third parties to obtain, verify and record information that identifies each person who opens a card account. What this means for you: When you open a card account, we will ask for your name, address, date of birth, social security number or country identification number, and other information that will allows us to identify you. We may also request a copy of your driver’s license or other documentation bearing your photo as verification of your identity.
          <br />
          Apple and the Apple logo are trademarks of Apple Inc., registered in the U.S. and other countries. App Store is a service mark of Apple Inc. Google Play is a trademark of Google Inc. Data rates may apply. Samsung Pay is a trademark of Samsung Inc.
        </div>
      </div>
    </div>
    <footer
      className="container mb-3"
    >
      <div
        className="d-none d-md-block"
      >
        <div
          className="d-flex justify-content-between border-bottom"
        />
        <div
          className="d-flex justify-content-between border-top"
        >
          <div
            className=""
          >
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/personal-profile"
              >
                dev docs
              </a>
            </div>
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Lore
              </a>
            </div>
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Ipsum
              </a>
            </div>
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Nullam Atoidio
              </a>
            </div>
          </div>
          <div
            className="copyright mt-1"
          >
            <span>
              © 2021 Lorem ipsum dolor sit amet, consectetuer
            </span>
          </div>
        </div>
      </div>
      <div
        className="d-block d-md-none"
      >
        <div
          className="d-flex border-bottom"
        >
          <div
            className="d-flex flex-column me-2"
          >
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/personal-profile"
              >
                dev docs
              </a>
            </div>
            <div
              className="footer-link mb-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Lore
              </a>
            </div>
            <div
              className="footer-link mb-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Ipsum
              </a>
            </div>
            <div
              className="footer-link mb-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Nullam Atoidio
              </a>
            </div>
          </div>
        </div>
        <div
          className="d-flex justify-content-end border-top"
        >
          <div
            className="copyright mt-2 text-end small"
          >
            <span>
              © 2021 Lorem ipsum dolor sit amet, consectetuer
            </span>
          </div>
        </div>
      </div>
    </footer>
  </div>,
]
`;
